pkg/cli/cmd_helpers.go

```diff
@@ -285,30 +285,22 @@ func (factory *executionHooksFactory) preRunEFunc(
 		}
 
 		// Pre-scaffold hook.
-		if err := factory.forEach(func(subcommand plugin.Subcommand) error {
+		return factory.forEach(func(subcommand plugin.Subcommand) error {
```

Comment:
`revive` complains about these checked, but always returned errors. 

This check can be turned off, if that would be desired, by adding this to the `linters-settings:` section:

```
  revive:
    rules:
      - name: if-return
        disabled: true
```

Reply:
Yes turn this off.

Reply:
Done.

I also reverted all the associated code changes.